src/app/GitCommands/Logging/CommandLog.cs

```diff
@@ -92,28 +92,49 @@ internal CommandLogEntry(string fileName, string arguments, string workingDir, D
             IsOnMainThread = isOnMainThread;
         }
 
+        internal const string WslGitLogName = "~git";
+        internal const string NativeGitLogName = "git";
+
         public string ColumnLine
         {
             get
             {
+                string wslCmd = AppSettings.WslCommand;
+                string wslGitCmd = AppSettings.WslGitCommand;
+                string gitCmd = AppSettings.GitCommand;
+
                 string duration = Duration is null
                     ? "running"
                     : $"{((TimeSpan)Duration).TotalMilliseconds:0,0} ms";
 
                 string fileName;
                 string arguments;
-                if (FileName.StartsWith("wsl "))
+                if (FileName.StartsWith(wslCmd) && FileName.EndsWith(wslGitCmd))
                 {
-                    fileName = "wsl";
+                    // WSL git commands from `GitModule`
+                    // Expect the "wsl" command, the wsl arguments and the "git" command in `FileName` and only the git command arguments in `Arguments`.
```

Comment:
I'm having troubles parsing this comment, do you think this could be reworded?
The key info to provide _why the filename is set to "~git"_.

Also, an example could be very useful. E.g.,
```
// FileName = "~git"
// Arguments = "...."
```